### PR TITLE
[WFLY-13439] - CVE-2020-6950 jsf-impl: Mojarra: Path traversal via ei…

### PR DESCRIPTION
**Pull request by adamecp**
ENTSWM-771 - CVE-2020-6950 jsf-impl: Mojarra: Path traversal via either the loc parameter or the con parameter, incomplete fix of CVE-2018-14371 [thorntail-2]

EAP Issue: https://issues.redhat.com/browse/ENTSWM-771

### CODE REVIEW

**Comment by bstansberry** [comment]
Hi @adamecp -- thanks for the test!

Can you file a WFLY JIRA and reference it in the commit message and PR title instead of ENTSWM?  ENTSWM is not meaningful to WildFly community and I believe it's not a public JIRA project.

@fjuma please review

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/testurl/ForbiddenUrlTestCase.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+/**
+ * UnallowedUrlTestCase
+ * For more information visit <a href="issues.jboss.org/browse/ENTSWM-771">https://issues.redhat.com/browse/ENTSWM-771</a>
```
Comment (fjuma) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/testurl/ForbiddenUrlTestCase.java:25]: This should reference a WFLY issue.

**Review by fjuma — APPROVED** [review]
Marking this one as approved since the test looks good. The commit message and PR title should be updated as Brian mentioned. Thanks @adamecp!

**Comment by adamecp** [comment]
@bstansberry , @fjuma Ok. I changed ENTSWM-771 to WFLY-13439

**Pull request merged**